bindings/yarp.i

```diff
@@ -306,6 +306,8 @@ void setExternal2(yarp::sig::Image *img, PyObject* mem, int w, int h) {
 %enddef
 %define YARP_init_API
 %enddef
+%define YARP_OS_DEPRECATED_API
```

Comment:
The compilation fails otherwise with a "Swig input error" (or something like that). As far as I understand, Swig can't follow the #define of `YARP_OS_DEPRECATED_API` all the way down to where it is actually defined, as it is compiler dependent. I guess it's the same reason as for all the other `%define` lines above.